crypto/x509/t_x509.c

```diff
@@ -65,10 +65,8 @@ int X509_print_ex(BIO *bp, X509 *x, unsigned long nmflags,
         nmindent = 12;
     }
 
-    if (nmflags == X509_FLAG_COMPAT) {
-        nmindent = 16;
```

Comment:
Same as the above comment, its unclear if the original intent was to check against `cflag` instead of `nmflags`, given the other values for the enumerations, it seems likely the the right side, rather than the left side of the equality is the incorrect one.  In any case, its been testing against `nmflags` since the code was introduced in 2000, and under that condition (nmflags == 0), the value of `nmindent` has been irrelevant for almost as long. (See also https://github.com/openssl/openssl/issues/18004),